src/tests/topp/IDFilter_5b_output.idXML

```diff
@@ -23,6 +21,8 @@
 			</PeptideHit>
 			<PeptideHit score="11.38" sequence="FINFGVNVEVLSRFQTK" charge="2" >
 			</PeptideHit>
+			<PeptideHit score="11.37" sequence="UFF" charge="2" protein_refs="PH_0" >
```

Comment:
The hits are now sorted by score. This happens in `IDFilter::assignRanks`, which before was only called for some filters (but possibly multiple times), but now always gets called once in the end. I think that makes the result more predictable.